vnext/PropertySheets/Bundle.props

```diff
@@ -50,14 +50,17 @@
     <BundleEntryFile Condition="'$(BundleEntryFile)' == ''">index.js</BundleEntryFile>
 
     <!-- Optional config file for the bundle command (metro will use metro.config.js by default) -->
-    <BundleConfigFile Condition="'$(BundleConfigFile)' == ''"></BundleConfigFile>
+    <BundleConfigFile Condition="'$(BundleConfigFile)' == ''"/>
```

Comment:
>BundleConfigFile [](start = 5, length = 16)

Why is it needed? It says: if BundleConfigFile was empty then make it empty. The net result is the same as we just remove this line.

Reply:
This part of the change (changing it to self-closing tag) was made by VS. I think the original intent of the line was to express that this is a property that can be set.

Reply:
Having said that, nothing is actually consuming this flag (see #5912) so removing this flag